### DISCUSSION TITLE
Use case

### DISCUSSION THREAD

**Discussion by polyGeek** [Yukams/background_locator_fixed#24] [General]
I've read the documentation and I _think_ I understand how this works, but I'd appreciate it if you could give me a yes/no if this use case is covered. I created the RunPee app: the app that tells you the best time to run and pee during a movie so you don't miss the best scenes. Obviously, this app is intended to be used at movie theaters. What I would like to do is add the option for users to get a notification when they are in the vicinity of the movie theater, in case they forget to use the app. Would this package make that possible?

From what I gather it will survive a reboot on Android, but not iOS, so that would be a limitation. It would be ideal if I could use this package to automatically restart the RunPee app.

**Comment by Yukams** [comment]
Hi @polyGeek this is a very interesting use case for this library. I guess it would absolutely be possible as long as you have a database with all locations for every movie theater, not sure that it would be an easy task though.
You're right about iOS, it seems that the original author couldn't make it able to start automatically after a phone restart. 
I don't know if this limitation is due to iOS itself or due to the author's knowledge. Though I'm not an iOS developer and I can't provide a solution to your problem.
Still, you could maybe use the library and improve it if you think you're able to do it !

I hope you'll find a way to make your project idea :)

**Reply by polyGeek** [reply]
@Yukams Thanks. I'll try to work this feature into the next update and report back how it goes.

**Reply by Yukams** [reply]
Awesome ! I believe that if you need help, you could fill an issue and someone from the community could lend you a hand ! Keep us updated :)
